src/Query/Builder.php

```diff
@@ -84,7 +84,7 @@ public function orderByDesc($column): Builder
     /**
      * Enhanced pagination is not possible as count query is not possible in datastore.
      */
-    public function paginate($perPage = 15, $columns = ['*'], $pageName = 'page', $page = null)
+    public function paginate($perPage = 15, $columns = ['*'], $pageName = 'page', $page = null, $total = null)
```

Comment:
Why you have added `$total = null` here? What's the use of it?

Reply:
```
Method 'Appsero\LaravelDatastore\Query\Builder::paginate()' is not compatible with method 'Illuminate\Database\Query\Builder::paginate()'.intelephense(P1038)
```

Reply:
oops. In 11, laravel changed their signature.